src/objects/shared-function-info-inl.h

```diff
@@ -672,6 +690,34 @@ void SharedFunctionInfo::ClearBinAstParseData() {
   DCHECK(!HasUncompiledDataWithBinAstParseData());
 }
 
+void SharedFunctionInfo::ClearInnerBinAstParseData() {
+  DCHECK(HasUncompiledDataWithInnerBinAstParseData());
+  UncompiledDataWithInnerBinAstParseData data = uncompiled_data_with_inner_bin_ast_parse_data();
+
+  // Trim off the pre-parsed scope data from the uncompiled data by swapping the
+  // map, leaving only an uncompiled data without pre-parsed scope.
```

Comment:
Is this comment accurate?

Reply:
heh, no looks like its not. i copied from ClearBinAstParseData, which looks like it came from being copied from ClearPreparseData? and it seems inaccurate there too... will remove both

Reply:
Ha, whoops. Copy paste coding strikes again :sweat_smile: